package.json

```diff
@@ -21,11 +21,14 @@
     "babel-preset-taskcluster": "^3.0.0",
     "babel-runtime": "^6.22.0",
     "debug": "^2.6.0",
+    "ip": "^1.1.4",
+    "ipaddr.js": "^1.2.0",
     "lodash": "^4.17.4",
     "mz": "^2.6.0",
     "promise": "^7.1.1",
     "pulse-publisher": "^1.1.4",
     "slugid": "^1.1.0",
+    "source-map-support": "https://registry.npmjs.org/source-map-support/-/source-map-support-0.2.10.tgz",
```

Comment:
I typed `npm install --save source-map-support` and this is what it did, I'm not sure why.  It doesn't seem harmful.  Happy to move to just `^0.2.10`.  I suspect it's related to using a version <1.